### DISCUSSION TITLE
There is a way to drop DRM master in VT-2 (without switching to ChromeOS UI) now.

### DISCUSSION THREAD

**Discussion by supechicken** [dnschneid/crouton#5054] [Ideas]
Comes an idea after reading #5013 and #5051 :)

As the title says, dropping DRM master in Frecon is possible now by printing an escape sequence to the active VT-2 instance:

> 
> ### DRM Drop Master
> An escape code can be used to tell frecon to drop DRM master. This is useful when frecon is invoked with --wait-drop-master.
> 
> Example:
> ```shell
> printf "\033]drmdropmaster\a" > /run/frecon/current
> ```

... If someone can integrate this logic into `src/freon.c`, an X server can be started in VT-2 directly and workarounds like `ssh+sudo` on `crosh` are no longer required.

**Comment by dnschneid** [comment]
Nice find!  Looks like that was [added early last year](https://chromium.googlesource.com/chromiumos/platform/frecon/+/0e6b5ea5a6e9c14daec3f75b0e6d5dbdbf85b9d0).  Adding it to frecon.c would make sense but someone needs to test that it doesn't break things on older ChromeOS versions (for EOL'd devices).
